src/utils/package-json.ts

```diff
@@ -0,0 +1,192 @@
+import { resolve } from 'path'
+
+export function findPackageJSON (): {
+  version: string
+  dependencies: { [key: string]: string }
+  devDependencies: { [key: string]: string }
+} {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const pkg = require(resolve('package.json'))
+  return {
+    version: pkg.version,
+    dependencies: pkg.dependencies,
+    devDependencies: pkg.devDependencies
+  }
+}
+
+export function computePackageJSON (answer: any): string {
+  const pkg = findPackageJSON()
+  const template: any = {}
+  template.name = answer.name
+  template.main = answer.language === 'JavaScript' ? 'app.js' : 'dist/app.js'
+  template.scripts = computeScripts(answer)
+  template.dependencies = computeDependencies(answer, pkg)
+  template.devDependencies = computeDevDependencies(answer, pkg)
+  if (String(answer.lint).includes('eslint')) {
+    template.eslintConfig = computeESLintConfig(template.devDependencies)
+  }
+
+  return JSON.stringify(template, null, 2)
+}
+
+function sort (dependencies: { [key: string]: string }): { [key: string]: string } {
+  const keys = Object.keys(dependencies).sort()
+  const obj: { [key: string]: string } = {}
+  for (const key of keys) {
+    obj[key] = dependencies[key]
+  }
+  return obj
+}
+
+export function computeScripts (answer: any): { [key: string]: string } {
+  const scripts: any = {}
+  // Lint Command
+  if (answer.lint === 'standard') {
+    scripts.lint = 'standard'
+  }
+  if (answer.lint === 'ts-standard') {
+    scripts.lint = 'ts-standard'
+  }
+  if (String(answer.lint).includes('eslint')) {
+    scripts.lint = `eslint . ${answer.language === 'TypeScript' ? '--ext .ts' : ''}`
+  }
+
+  // Other Commands
+  if (answer.language === 'JavaScript') {
+    scripts.test = 'tap "test/**/*.test.js"'
+    scripts.start = 'fastify start -l info app.js'
+    scripts.dev = 'fastify start -w -l info -P app.js'
+  }
+  if (answer.language === 'TypeScript') {
+    scripts.test = 'tap "test/**/*.test.ts" --ts'
+    scripts.start = 'fastify start -r ts-node/register -l info app.js'
```

Comment:
is this supposed to be prod-ready start? should it be using ts-node then? no command to build & run?

Reply:
Yes, it needs `build` command.
It is doubt-able the needs for running in `node` directly because some of the detail for starting server is hide inside the CLI.